improv/store.py

```diff
@@ -53,7 +53,7 @@ class PlasmaStore(StoreInterface):
     def __init__(
         self,
         name="default",
-        store_loc="/tmp/store",
```

Comment:
I dislike getting rid of a useful default. Defaulting to "" is going to fail, so that can't be the right solution. I'd prefer to keep `"/tmp/store"` as a default (or create a unique one using `uuid` here).

Reply:
We could even create the uuid here and have Nexus get it from the returned object as `store.store_loc`. That hides the details of uuid creation inside the `Store` class itself.

Reply:
Conversely, if `Store` is really just a store interface, we do need to always pass it the location, but `createStore` above can just create the uuid and set it as an attribute.